### DISCUSSION TITLE
Automapper 11.0

### DISCUSSION THREAD

**Discussion by Erythnul** [LuckyPennySoftware/AutoMapper#3754] [NA]
Hi!

Wondering when 11.0 will be considered final and will be released as it contains a fix for an issue I reported and has been in development for quite some time.
I'd prefer not to use the Myget build in production code, despite its relative stability.

Kind regards,

Steven.

**Comment by jbogard** [comment]
It's probably about time for a release. Was considering waiting for .NET 6 but perhaps we can release ahead of it.

**Reply by Erythnul** [reply]
Thanks for the quick response, looking forward to it.

**Reply by magneticcore** [reply]
Will be great to have the version 11 for .NET 5, before the coming of .NET 6

**Reply by craigfowler** [reply]
> .NET 6 is upon you, whether you would risk it or not 😄

**Reply by killergege** [reply]
.NET 6 is there, any plan for a release ? ;)

**Reply by jbogard** [reply]
AutoMapper is already supported for .NET 6, .NET 5, .NET Core 3.1/0 etc. It's `netstandard2.0` which is forward-compatible from `netcore2.0` onwards.

**Reply by killergege** [reply]
Yes it is, My question wasn't really about .Net 6, more about version 11 :)
There are fixes that would be useful for handling record (for instance https://github.com/AutoMapper/AutoMapper/discussions/3629)

**Reply by magneticcore** [reply]
ho,ho,ho. I asked Santa Claus for this year to have version 11 as a Christmas present :smiley:

**Comment by jbogard** [comment]
I'm thinking of targeting only `netstandard2.1` for v11, which would be the last 3 versions of .NET (Core) and the last two LTS. This would then drop .NET Framework completely. Which I am fine with.

**Reply by lbargaoanu** [reply]
Sounds good :)

**Reply by craigfowler** [reply]
I'm of two minds.  My first instinct is to say "please don't drop .NET framework support yet".  In my day job I have a solution which still has two .NET Framework (MVC web app) projects which are yet to be migrated up.  If AutoMapper support were dropped for `netstandard2.0` and .NET Framework then - say we found an issue - I'd be forced to ask for back-patches of fixes to 10.x.  Of course there's a risk that neither you nor anybody else on the project are willing to do that.

On the other hand, if support were dropped then it will help me express to our product owner how urgent scheduling that migration is.  _"The .NET open source ecosystem is leaving us behind and we are left stuck on unsupported versions of libraries"_ etc.

I suppose _my real opinion_ is to ask (rhetorically) what it's costing you in practical terms by continuing to support `netstandard2.0`/.NET Framework.  If there are a heap of new tech in `netstandard2.1` you desperately wish to adopt, which would allow great improvements to AutoMapper, then I suppose so be it - upgrade away.  But if you feel like `netstandard2.0` isn't holding you back that much then I'd say "please keep support for now".

**Comment by jbogard** [comment]
AutoMapper 11.0 is out now, release notes [here](https://github.com/AutoMapper/AutoMapper/releases/tag/v11.0.0)

**Comment by vpetrusevici** [comment]
Hello guys! Do you have plans to update AutoMapper.Collection too?

**Reply by jbogard** [reply]
Question for @TylerCarlson1, he's the main maintainer of that one
